packages/flutter/lib/src/material/material.dart

```diff
@@ -200,15 +201,17 @@ class Material extends StatefulWidget {
   /// {@template flutter.material.material.elevation}
   /// The z-coordinate at which to place this material relative to its parent.
   ///
-  /// This controls the size of the shadow below the material.
+  /// This controls the size of the shadow below the material and the dark
+  /// theme overlay color if inside a dark theme.
```

Comment:
We're saying "dark theme" quite a bit, however from the API's point of view it's a theme for which `brightness` is `Brightness.dark`. It would be good to draw attention to that somewhere.